src/GtkUtilities.jl

```diff
@@ -44,7 +44,7 @@ include("panzoom.jl")
 using .PanZoom
 import .PanZoom: interior, fullview   # for extensions
 
-function Base.copy!{C<:Colorant}(ctx::CairoContext, img::AbstractArray{C})
+function Base.copy!{C<:Union{Colorant,Number}}(ctx::CairoContext, img::AbstractArray{C})
```

Comment:
Yes but by that logic any package can go ahead and define arbitrary additional operations on the types of any other package, as long as it's not overriding anything. There's a point at which that starts becoming a problem though, if someone introduces semantics that are controversial or have side effects more serious than error vs. not error. For non-controversial, obvious only ways things could work, my only concern is the invisible dependency issue. Maybe we need better automatic introspection tools on where the definitions of all called methods were defined, and tie that to the package dependency system at a method level.